app/(auth)/logout/actions.ts

```diff
@@ -0,0 +1,13 @@
+'use server';
+
+import { cookies } from 'next/headers';
+
+export async function logout() {
+  const fakeSessionToken = cookies().get('fakeSession');
+
+  if (!fakeSessionToken) return undefined;
+  // FIXME: Delete the session from the database
+
+  // set the cookie to be expired
+  await cookies().set('fakeSession', '', { maxAge: -1 });
+}
```

Comment:
Server action for logout, better to allow Next.js to handle the logout instead of creating a GraphQL mutation just for that.